### PR TITLE
Implement file download upload with S3.

### LINKED ISSUE #5: Implement file download / upload with S3

### PR DESCRIPTION
**Pull request by sukui24**
closes #5

### CODE REVIEW

File: server/compose.yml [hunk 1]
```diff
@@ -20,6 +20,20 @@ services:
...
       - pgadmin:/var/lib/pgadmin
 
+  # Url: http://localhost:9000/ and console on http://localhost:9001/
+  minio:
+    image: quay.io/minio/minio
+    container_name: minio
+    ports:
+      - "9000:9000"
+      - "9001:9001"
+    environment:
```
Comment (Asion001) [inline server/compose.yml:30]: Вище env а тут сразу?

File: server/src/services/ContentStructureService.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import { ValueUtils } from "../utils/ValueUtils";
+
+/**
+ * Class that manages all actions related to content.json file and it's structure
+ */
+export class ContentStructureService {
+  /**
+   * Parse content.json file and update all "file" entries with download link
+   */
+  public static async addLinksForFiles(
```
Comment (Asion001) [inline server/src/services/ContentStructureService.ts:13]: Do not need that

File: server/compose.yml (hunk 1 above)
Comment (sukui24) [reply server/compose.yml:30]: Da

File: server/src/services/ContentStructureService.ts (hunk 2 above)
Comment (sukui24) [reply server/src/services/ContentStructureService.ts]: Changed

**Pull request merged**